rollup.config.js

```diff
@@ -7,10 +7,10 @@ import pkg from './package.json';
 export default [
   // Brwoser-friendly UMD build
   {
-    input: 'src/main.js',
+    input: 'src/index.js',
     output: {
       name: 'nugit-datetime-utils',
-      file: pkg.browser,
+      file: 'lib/nugit-datetime-utils.umd.js',
```

Comment:
Why not keep using the `pkg` for file paths?

Reply:
Because `pkg.browser` is now set to esm version

Reply:
Isn't there another attribute of `pkg` we can use?

Reply:
`module` and `browser` have the same value, is there no other way?

Reply:
Actually `browser` field is not needed as it's same as `module`